apps/lfx-one/src/app/modules/events/my-events-dashboard/components/event-selection/event-selection.component.scss

```
@@ -0,0 +1,32 @@
+// Copyright The Linux Foundation and each contributor to LFX.
+// SPDX-License-Identifier: MIT
+
+.events-scroll-area {
+  @apply max-h-[340px];
+  scrollbar-width: thin;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)